test/partition-healing-tests.js

```diff
@@ -307,8 +309,8 @@ function decreaseIncOfSutInB(t, tc) {
 }
 
 
-function splitMemberships(t, tc) {
-    return function splitMemberships(list, cb) {
+function separateMemberships(t, tc) {
```

Comment:
Still not too happy with this. What do you think of localizeMemberships, since that is the word you use in the comment to describe the call .